alpha/alpha-server/src/main/resources/application.yaml

```diff
@@ -17,6 +17,16 @@
 server:
   port: 8090
 
+alpha:
```

Comment:
If we use the default setting, we don't need to specify it in the application.yaml.

Reply:
The settings here are for demonstration purposes.